src/containers/home/HomePage.js

```diff
@@ -27,133 +35,169 @@ import {
-          <GraphicShot>
-            <RightPeopleGraphic />
-          </GraphicShot>
           <Tile>
-            <Tile gap={16}>
-              <Typography color={PURPLE.P300} fontWeight={700} variant="body2">
-                RIGHT PEOPLE
-              </Typography>
-              <Typography maxWidth={{ lg: 400 }} variant="h2">
-                Put the data in the hands of people who need it
-              </Typography>
-            </Tile>
-            <Typography maxWidth={{ lg: 480 }} variant="body2">
-              Set access to data according to fine-grained permissions and roles.
+            <Typography maxWidth={{ lg: 576 }} variant="h2">
+              Collect, Analyze and Advance Your Research with Chronicle.
+            </Typography>
+          </Tile>
+          <Tile>
+            <Typography maxWidth={{ lg: 528 }} variant="body2">
+              We’re creating new ways for academic researchers to collect, analyze and advance their research with
+              Chronicle - a modern  Study Management tool that expands research capabilities. Chronicle enables
+              researchers and primary investigators to better understand participant behavior and unlocks new insights
+              with mobile functionality, ecological momentary assessment features, and customizable study management
+              workflows.
             </Typography>
-            <Typography maxWidth={{ lg: 480 }} variant="body2">
-              See who accessed what data and when.
-            </Typography>
-            <Tile gap={32}>
-              <Typography maxWidth={{ lg: 480 }} variant="body2">
-                OpenLattice makes sure only those who should have access to certain data actually do.
-              </Typography>
-              <SolidButton onClick={openBeacon}>Get a Demo</SolidButton>
-            </Tile>
           </Tile>
         </FeatureGrid>
       </SectionContent>
-      <SectionContent>
+      <SectionContent margin={{ b: 0, t: 100 }}>
+        <TileGrid>
+          <Tile align={{ h: 'start' }}>
+            <DBSyncCircleIcon width="50" />
+            <Typography textAlign="left" variant="h6">
+              Discover More
+            </Typography>
+            <Typography textAlign="left" variant="body1">
+              Enable participants to share data through a simple mobile application and access new streams of
+              information, such as time use diaries and 3rd party sensors.
+            </Typography>
+            <LearnMoreButton href={PRODUCTS_CHRONICLE} />
+          </Tile>
+          <Tile align={{ h: 'start' }}>
+            <LinkCircleIcon width="50" />
+            <Typography textAlign="left" variant="h6">
+              Real-time Insights
+            </Typography>
+            <Typography textAlign="left" variant="body1">
+              {`Advance your research using our ecological momentary assessment features to measure participant's
+              behaviors and experiences in real-time.`}
```

Comment:
Can you make sure the new line here isn't being rendered in the browser?

Reply:
the new line is not being rendered in the browser!